ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/domain/CaseEvent.java

```diff
@@ -58,7 +58,8 @@
     //inversion of control
     INITIATE_CLAIM_PAYMENT_CITIZEN("InitiateClaimPaymentCitizen"),
     RESUME_CLAIM_PAYMENT_CITIZEN("ResumeClaimPaymentCitizen"),
-    CREATE_LEGAL_REP_CLAIM("CreateLegalRepClaim");
+    CREATE_LEGAL_REP_CLAIM("CreateLegalRepClaim"),
+    CREATE_CITIZEN_CLAIM("CreateCitizenClaim");
```

Comment:
This is done as part of @simad 's story - ROC-6121. I just added it here for use in my PR. I will merge and replace this changes from her PR once it's merged to master.